packages/offix-datastore/src/replication/documents/buildGraphQLCRUDMutations.ts

```diff
@@ -0,0 +1,49 @@
+import gql from "graphql-tag";
+
+import { Model } from "../../Model";
+import { ReplicatorMutations } from "./Documents";
+
+/**
+ * Builds GraphQL mutations for models following
+ * the [GraphQLCRUD specification]{@link https://graphqlcrud.org/}
+ */
+export const buildGraphQLCRUDMutations = (models: Model[]): Map<string, ReplicatorMutations> => {
```

Comment:
Since we can have a situation where we do not need subscriptions etc. so I have split creation into separate method for mutations, subscriptions and deltas. Those are build based on the config.